plutus-tx/testlib/PlutusTx/Test.hs

```diff
@@ -259,9 +282,29 @@ runPlcCek values = do
   ps <- traverse toUPlc values
   let p = foldl1 (unsafeFromRight .* UPLC.applyProgram) ps
   fromRightM (throwError . SomeException) $
-    UPLC.evaluateCekNoEmit
-      PLC.defaultCekParametersForTesting
-      (p ^. UPLC.progTerm)
+        UPLC.evaluateCekNoEmit
+        PLC.defaultCekParametersForTesting
+        (p ^. UPLC.progTerm)
+
+runPlcCekBudget
+  :: (ToUPlc a PLC.DefaultUni PLC.DefaultFun)
+  => [a]
```

Comment:
We don't seem to use this list functionality anywhere? So can make it just `a`?

Reply:
I agree. It would be simpler to make this just `a` and have some other helper that does `ToUplc a => [a] -> a`.

Reply:
Yep, when I was implementing `evaluateCompiledCode` I simplified interface to accept a single argument. Users can chain infix applications externally.